libcnb-package/src/lib.rs

```diff
@@ -255,42 +255,37 @@ pub fn find_buildpack_dirs(start_dir: &Path, ignore: &[PathBuf]) -> std::io::Res
 
 /// Provides a standard path to use for storing a compiled buildpack's artifacts.
 #[must_use]
-pub fn get_buildpack_target_dir(
+pub fn get_buildpack_package_dir(
```

Comment:
`get_buildpack_target_dir` was public so this needs a changelog entry

Reply:
Added in [a3690ba](https://github.com/heroku/libcnb.rs/pull/583/commits/a3690ba54199288f6bc4f3ec44a96ec0f7e528ce)